package.json

```diff
@@ -4,10 +4,10 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "node server/main.js",
-    "build": "webpack --config ./webpack.config.js --progress",
-    "postinstall": "npm run build", 
-    "test": "echo \"Error: no test specified\" && exit 1"
+  "start": "node server/main.js",
+  "build": "./node_modules/webpack/bin/webpack.js -w -c",
+  "postinstall": "node server/database/worker/dbgrabber.js && npm run build",
```

Comment:
Any reason why you're going back to rebuilding the database every time the server is run?